app/assets/javascripts/discourse/templates/preferences/users.hbs

```diff
@@ -0,0 +1,29 @@
+{{#if siteSettings.ignore_user_enabled}}
+  <div class="control-group muting">
+    <label class="control-label">{{i18n 'user.users'}}</label>
+    <div class="controls category-controls">
+      <label>{{d-icon "d-muted" class="muted icon"}} {{i18n 'user.muted_users'}}</label>
+      {{user-selector excludeCurrentUser=true usernames=model.muted_usernames class="user-selector"}}
+    </div>
+    <div class="instructions">{{i18n 'user.muted_users_instructions'}}</div>
+
+    <div class="controls category-controls">
+      <label>{{d-icon "eye-slash" class="muted icon"}} {{i18n 'user.ignored_users'}}</label>
```

Comment:
do we "muted icon" in css somewhere ?

Reply:
Good catch. We do use it in the top muted users `d-icon`, I don't see `muted` class defined or used anywhere else. I will remove it 👍